src/Sentry/SentryOptions.cs

```diff
@@ -785,6 +789,15 @@ public void AddJsonConverter(JsonConverter converter)
     [EditorBrowsable(EditorBrowsableState.Never)]
     public INetworkStatusListener? NetworkStatusListener { get; set; }
 
+    /// <summary>
+    /// Allows integrations to provide a custom assembly reader.
+    /// </summary>
+#if !__MOBILE__
+    [CLSCompliant(false)]
+#endif
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public Func<string, PEReader?>? AssemblyReader { get; set; }
+
```

Comment:
This is the new extension point for connecting the reader, which we do automatically for `Sentry/net6.0-android` and we can use again in `Sentry.Xamarin`.